adafruit_rfm9x.py

```diff
@@ -873,8 +752,8 @@ def receive(
         # Enter idle mode to stop receiving other packets.
         self.idle()
         if not timed_out:
-            if self.enable_crc and self.crc_error:
-                warn("CRC error, packet ignored")
+            if self.enable_crc and self.crc_error():
+                self.crc_error_count += 1
```

Comment:
@jerryneedell  q: with `warn` removed, user will not be notified of a CRC error. Would this be handled in user-code?

Reply:
yes -- something like this 
```
crc_count = 0
while True:
    # Look for a new packet: only accept if addresses to my_node
    packet = rfm9x.receive(with_ack=True, with_header=True)
    if rfm9x.crc_error_count != crc_count:
        crc_count = rfm9x.crc_error_count
        print("CRC error ",crc_count)
```

Reply:
In many cases -- user may not care or want to know -- the packet will just be ignored